packages/trpc/server/routers/viewer/teams/removeMember.handler.ts

```diff
@@ -28,7 +28,10 @@ export const removeMemberHandler = async ({ ctx, input }: RemoveMemberOptions) =
   });
 
   const { memberIds, teamIds, isOrg } = input;
-  const isAdmin = teamIds.every(async (teamId) => await isTeamAdmin(ctx.user.id, teamId));
```

Comment:
This is the wrong way we handled it before the promise inside every. `isAdmin` is set to true before